typeid/typeid-go/typeid.go

```diff
@@ -6,42 +6,58 @@ import (
 )
 
 // TypeID is a unique identifier with a given type as defined by the TypeID spec
-type TypeID[P PrefixType] struct {
-	prefix string
-	suffix string
+type TypeID struct {
+	value     string // The complete "prefix_suffix" string
```

Comment:
Is the suffix the encoded bytes? Maybe just add a comment to clarify.

Reply:
Doing.